### PR TITLE
Fix: in PHP 8.0 method NumericHelper::normalize throws error for floa…

### PR DESCRIPTION
**Pull request by vjik**
…t or int values.

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

Fixed error (throws in PHP8):

```
NumericHelper::normalize(10.0); // TypeError: str_replace(): Argument #3 ($subject) must be of type array|string, float given
```

### CODE REVIEW

File: tests/NumericHelperTest.php [hunk 1]
```diff
@@ -37,19 +37,21 @@ public function testToOrdinalWithIncorrectType(): void
     public function normalizeNumberDataProvider(): array
     {
         return [
-
             'French' => ['4 294 967 295,000', '4294967295.000'],
             'German' => ['4 294 967.295,000', '4294967295.000'],
             'Spanish' => ['4.294.967.295,000', '4294967295.000'],
             'English' => ['4,294,967,295.000', '4294967295.000'],
             'Smaller' => ['10,111', '10.111'],
+            [10.0, '10'],
```
Comment (rustamwin) [inline tests/NumericHelperTest.php:45]:
```suggestion
            'Float' => [10.0, '10'],
```
or something like this

Comment (vjik) [reply tests/NumericHelperTest.php]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ## 1.1.1 under development
 
 - Enh #40: Use `str_starts_with()` and `str_ends_with()` if available (viktorprogger)
+- Fix: in PHP 8.0 method `NumericHelper::normalize` throws error for `float` or `int` values (vjik)
```
Comment (rustamwin) [inline CHANGELOG.md:6]:
```suggestion
- Bug #43: `NumericHelper::normalize` throws an error for `float` or `int` values in PHP 8 (vjik)
```

**Review by rustamwin — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (vjik) [reply CHANGELOG.md]: Thanks!

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**